docs/architecture/04-loop-execution-flow.html

```
@@ -0,0 +1,269 @@
+<rect x="1300" y="540" width="140" height="60" rx="6" class="c-proc" stroke-width="1.5"/>
+<text x="1370" y="565" class="t-primary" font-size="12" font-weight="600" text-anchor="middle">更新 phase</text>
+<text x="1370" y="583" class="t-muted" font-size="9" text-anchor="middle">生命周期</text>
+
+<!-- 门禁类型说明 -->
+<rect x="100" y="630" width="1360" height="100" rx="6" fill="rgba(251,191,36,.08)" stroke="#fbbf24" stroke-width="0.5" stroke-dasharray="3 2"/>
+<text x="112" y="652" class="t-muted" font-size="10" font-weight="600">门禁类型（gate_config）</text>
+<text x="112" y="672" class="t-muted" font-size="9">• ai_criteria_review：AI 评分门禁，min_score 阈值</text>
+<text x="112" y="690" class="t-muted" font-size="9">• human_approval：人工审批门禁，pending → 审批推进 passed/failed</text>
+<text x="112" y="708" class="t-muted" font-size="9">• 废弃类型 artifact_present / script_check 已统一并入 ai_criteria_review</text>
+<text x="700" y="672" class="t-muted" font-size="9">• 流转策略：next / skip / end / break / 裸 step_id（goto 跳转）</text>
+<text x="700" y="690" class="t-muted" font-size="9">• 返工判定：target_idx <= current_idx → rework_count++，超 max_rework 强制失败</text>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)